apps/etterlatte-trygdetid/src/main/kotlin/trygdetid/klienter/KodeverkKlient.kt

```diff
@@ -0,0 +1,36 @@
+package no.nav.etterlatte.trygdetid.klienter
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties
+import com.typesafe.config.Config
+import io.ktor.client.HttpClient
+import io.ktor.client.call.body
+import io.ktor.client.request.accept
+import io.ktor.client.request.get
+import io.ktor.client.request.header
+import io.ktor.http.ContentType
+import org.slf4j.LoggerFactory
+import java.util.*
+
+class KodeverkKlient(config: Config, private val httpKlient: HttpClient) {
```

Comment:
Ville kanskje lagt til en klient-test for å sikre at deserialisering m.m. blir korrekt.

Reply:
Skal se på det 👍 Mulig det blir litt endring i neste PR.